samples/traffic_meter/line_crossing.py

```diff
@@ -104,41 +127,49 @@ def process_frame(self, buffer: Gst.Buffer, frame_meta: NvDsFrameMeta):
                     self.track_last_frame_num[frame_meta.source_id][
                         obj_meta.track_id
                     ] = frame_meta.frame_num
-                    direction = lc_tracker.check_track(obj_meta.track_id)
 
-                    obj_events = self.cross_events[frame_meta.source_id][
-                        obj_meta.track_id
-                    ]
-                    if direction is not None:
-                        # send to graphite
-                        self.stats_client.incr(
-                            '.'.join(
-                                (
-                                    frame_meta.source_id,
-                                    self.target_obj_label,
-                                    direction.name,
-                                )
+                    obj_metas.append(obj_meta)
+
+            track_lines_crossings = lc_tracker.check_tracks(
+                tuple(obj_meta.track_id for obj_meta in obj_metas)
```

Comment:
I suggest avoiding unnecessary conversions. Can the method take a list?

Reply:
This is a tuple from a generator expression, there's no extra conversions involved, as far as I understand.

Changed the argument to a list comprehension.